tests/test_problem_bank_helpers_rounding.py

```diff
@@ -0,0 +1,221 @@
+from src.problem_bank_helpers import __version__
+from src.problem_bank_helpers import problem_bank_helpers as pbh
+from collections import defaultdict
+import numpy as np
+import sigfig
+import pandas as pd
+import os
+import pytest
+
+def test_version():
+    assert __version__ == '0.1.13'
+
+test_values = {"between0and1": 
+                    [0.1,
+                     0.12,
+                     0.123,
+                     0.1234,
+                     0.12345,
+                     0.123456,
+                     0.1234567,
+                     0.12345678,
+                     0.123456789,
+                     0.1234567891,
+                     0.12345678912,
+                     0.123456789123
+                    ],
+                "singleDigits":[
+                    0,
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    6,
+                    7,
+                    8,
+                    9
+                    ],
+                "leadingZeros":[
+                    0.1,
+                    0.012,
+                    0.00123,
+                    0.0001234
+                ],
+                "trailingZeros":[
+                    10,
+                    1200,
+                    123000,
+                    12340000,
+                    1234500000,
+                ],
+                "emptyDecimal":[
+                    "1.",
+                    "10.",
+                    "100.",
+                    "1000.",
+                    "10000.",
+                    "100000.",
+                    "1000000.",
+                    "10000000.",
+                    "100000000."
+                ],
+                "zeroDecimal":[
+                    "1.0",
+                    "1.00",
+                    "1.000",
+                    "1.0000",
+                    "1.00000",
+                    "1.000000",
+                    "1.0000000",
+                    "1.00000000",
+                    "1.000000000",
+                    "1.0000000000"
+                ],
+                "largeNumbers":[
+                    5,
+                    55,
+                    555,
+                    5555,
+                    11111,
+                    111111,
+                    1111111,
+                    11111111
+                ],
+                "eFormat":[
+                    "1e10",
+                    "1.1e10",
+                    "1.12e20",
+                    "1.123e30"
+                ]}
+
+group_names = list(test_values.keys())
+
+@pytest.fixture
+def variables():
+    inputs = test_values
+    return inputs
+
+# Test sigfigs function
+@pytest.mark.parametrize(
+    "test_group",
+    group_names
+)
+def test_sigfigs(test_group, variables):
+    for i in range (0, len(variables[test_group])):
+        test_input = str(variables[test_group][i])
```

Comment:
```suggestion
    for i, test_input in enumerate(variables[test_group]):
        test_input = str(test_input)
```
Suggestion: use `enumerate` over excessive indexing since we want both the item index and item